src/components/buttons/browse-the-book.cjsx

```diff
@@ -21,25 +21,23 @@ module.exports = React.createClass
     unstyled:  React.PropTypes.bool
     bsStyle:   React.PropTypes.string
 
-  getLinkProps: (link) ->
-    linkProps = {target:'_blank', className:'view-reference-guide', href: link}
+  getLinkProps: ->
+    linkProps =
+      className: 'view-reference-guide'
+
     linkProps.className += " #{@props.className}" if @props.className
-    text = @props.children or 'Browse the Book'
+    linkProps.className += " btn btn-#{@props.bsStyle}" unless @props.unstyled
 
```

Comment:
Hrm, can we maybe add classnames lib?  We're already including it in the build since react-bootstrap uses it internally.

+1 for renaming the component, it's not always a `Button`